crates/rpc-types-eth/src/state.rs

```diff
@@ -1,9 +1,10 @@
 //! bindings for state overrides in eth_call
 
 use crate::BlockOverrides;
+use alloc::boxed::Box;
 use alloy_primitives::{Address, Bytes, B256, U256};
+use hashbrown::HashMap;
```

Comment:
we could either use std hashmap or hashbrown because this change will break in a few places

Reply:
this is a breaking change